src/containers/StructurePage/resourceComponents/ResourceItems.tsx

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright (c) 2017-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { memo, useState } from 'react';
+import { injectT, tType } from '@ndla/i18n';
+import Resource from './Resource';
+import {
+  deleteTopicResource,
+  updateTopicResource,
+  updateTopicSubtopic,
+  updateSubjectTopic,
+} from '../../../modules/taxonomy';
+import handleError from '../../../util/handleError';
+import MakeDndList from '../../../components/MakeDndList';
+import AlertModal from '../../../components/AlertModal';
+import Spinner from '../../../components/Spinner';
+import { TopicResource } from './StructureResources';
+import { classes } from './ResourceGroup';
+
+interface Props {
+  resources: TopicResource[];
+  refreshResources: () => Promise<void>;
+  locale: string;
+}
+
+const ResourceItems = ({ refreshResources, resources, locale, t }: Props & tType) => {
+  const [deleteId, setDeleteId] = useState<string>();
+  const [error, setError] = useState<string>();
+  const [loading, setLoading] = useState<boolean>();
```

Comment:
Kanskje ikke så viktig, men synes det er bedre å initialisere disse. Hvis du hovrer over i vscode ser du at den egentlige typen blir `boolean | undefined`. Gjelder også flere steder

Reply:
Tenker du da å eksplisitt initialisere de som `undefined`?

Reply:
Nei, som false
```suggestion
  const [loading, setLoading] = useState<boolean>(false);
```
Kan også initialisere med tom streng på de over